### PR TITLE
Hughsie/synaptics mst oui

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
@YiJaneWu maybe this shows why the DPCD is so powerful :)

File: libfwupdplugin/fu-dpaux-device.c [hunk 1]
```diff
@@ -128,6 +128,33 @@ fu_dpaux_device_setup(FuDevice *device, GError **error)
...
+					 "OUI",
+					 "HWREV",
+					 NULL);
+	fu_device_build_instance_id_full(device,
+					 FU_DEVICE_INSTANCE_FLAG_QUIRKS,
+					 NULL,
+					 "DPAUX",
+					 "OUI",
+					 "DEVID",
+					 NULL);
```
Comment (superm1) [inline libfwupdplugin/fu-dpaux-device.c:155]: presumably all the DP aux plugins README should mention this too now

**Review by superm1 — DISMISSED** [review]
Looks great, just some doc improvement.

File: libfwupdplugin/fu-dpaux-device.c (hunk 1 above)
Comment (hughsie) [reply libfwupdplugin/fu-dpaux-device.c:155]: Do you think that we want a DPAUX\OUI+DEVID+HWREV combined one too?

Comment (hughsie) [reply libfwupdplugin/fu-dpaux-device.c:155]: Both bits added.

**Pull request merged**